packages/lesswrong/server/scripts/backfillParentTags.ts

```diff
@@ -0,0 +1,49 @@
+import TagRels from "../../lib/collections/tagRels/collection";
+import Tags from "../../lib/collections/tags/collection";
+import { Globals, updateMutator } from "../vulcan-lib";
+import { v4 as uuid } from 'uuid';
+import { updatePostDenormalizedTags } from "../tagging/tagCallbacks";
+
+const backfillParentTags = async (childTagSlug, parentTagSlug) => {
```

Comment:
Given that we have to do a lot of these it would be more useful to have it take the parent tag slug only, and iterate over all the child tags automatically